asv_bench/benchmarks/benchmarks.py

```diff
@@ -148,6 +193,38 @@ def time_merge(self, shapes, how, sort):
         )
 
 
+class TimeMerge2:
```

Comment:
Can we use a more descriptive name or just combine it with `TimeMerge`?

Reply:
Let me try this.

Reply:
looks good